spec/javascripts/support/jasmine.yml

```diff
@@ -17,7 +17,11 @@ css_dir: "app/assets/stylesheets"
 # list of file expressions to include as source files
 # relative path from src_dir
 src_files:
-  - "application.{js.coffee,js,coffee}"
+  - "../../../spec/javascripts/helpers/jquery-1.12.4.js"
+  - "../../../spec/javascripts/helpers/jasmine-jquery-2.0.5.js"
+  - "../../../spec/javascripts/helpers/cookie-functions.js"
```

Comment:
Where is this used? I see no other references to `cookie` in the spec that's added.